SpaceX/HomeScreen/HistoryView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  HistoryView.swift
+//  SpaceX
+//
+//  Created by Julia on 20.07.2022.
+//
+
+import UIKit
+
+class HistoryView: UIView {
+    
+    lazy var articleTitle: UILabel = {
+        let articleTitle = UILabel()
+        articleTitle.text = titleText
+        articleTitle.numberOfLines = 0
+        articleTitle.font = UIFont(name: "Inter-Regular", size: 20)
+        return articleTitle
+    }()
+    
+    lazy var articleLabel: UILabel = {
+        let articleLabel = UILabel()
+        articleLabel.text = articleText
+        articleLabel.numberOfLines = 0
+        articleLabel.font = UIFont(name: "Inter-Regular", size: 12)
+        return articleLabel
+    }()
+    
+    lazy var articleButton: UIButton = {
+        let articleLink = UIButton()
+        let yourAttributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont(name: "Inter-Regular", size: 12) as Any,
+            .underlineStyle: NSUnderlineStyle.single.rawValue
+        ]
+        let attributeString = NSMutableAttributedString(
+            string: "Read article in web...",
+            attributes: yourAttributes
+        )
+        articleLink.setAttributedTitle(attributeString, for: .normal)
+        return articleLink
+    }()
+    
+    lazy var chevronImage: UIImageView = {
+        let chevronImage = UIImageView(image: UIImage(named: "arrow")?.withTintColor(.sxItemColor))
+        return chevronImage
+    }()
+    
+    let titleText: String?
+    let articleText: String?
+    let link: String?
+    let homeVC: HomeViewController
```

Comment:
Why you create here HomeViewController?
Can we add action in HomeViewController instead of this view?